packages/strapi-admin/admin/src/translations/pt.json

```diff
@@ -1,173 +1,217 @@
+  "app.components.HomePage.createBlock.content.second": " extensão irá ajudar-te a definir a estrutura de dados dos teus modelos. Se és novo aqui, recomendamos fortemente que sigas o nosso ",
   "app.components.HomePage.createBlock.content.tutorial": " tutorial.",
-  "app.components.HomePage.cta": "CONFIRME",
-  "app.components.HomePage.newsLetter": "Subscreva-se à newsletter para ficar a par sobre Strapi",
+  "app.components.HomePage.cta": "CONFIRMAR",
+  "app.components.HomePage.newsLetter": "Subscreve-te à newsletter para ficares a par do Strapi",
   "app.components.HomePage.support": "SUPORTE-NOS",
-  "app.components.HomePage.support.content": "Comprando a camisola, permitirá-nos continuar o nosso trabalho no projecto para dar à si a melhor experiência possível!",
-  "app.components.HomePage.support.link": "OBTENHA A SUA CAMISOLA AGORA",
+  "app.components.HomePage.support.content": "Ao comprar a t-shirt, irá permitir-nos continuar o nosso trabalho no projeto para te proporcionar a melhor experiência possível!",
+  "app.components.HomePage.support.link": "OBTENHA A SUA T-SHIRT AGORA",
   "app.components.HomePage.welcome": "Bem-vindo(a) a bordo",
   "app.components.HomePage.welcome.again": "Bem-vindo(a) ",
-  "app.components.HomePage.welcomeBlock.content": "Estamos felizes em tê-lo como um dos membros da comunidade. Estamos constantemente a procura de feeback então sinta-se à vontade para enviar-nos uma mensagem em privado no ",
-  "app.components.HomePage.welcomeBlock.content.again": "Esperamos que estejas a progredir em seu projecto... SInta-se à vontade em ler as nossas últimas publicações sobre Strapi. Estamos dando o nosso melhor para amelhorar o produto baseando-se no seu feedback.",
+  "app.components.HomePage.welcomeBlock.content": "Estamos felizes em ter-te como um dos membros da comunidade. Estamos constantemente à procura de feedback, por isso sente-te à vontade para nos enviares uma mensagem em privado no ",
+  "app.components.HomePage.welcomeBlock.content.again": "Esperamos que estejas a progredir no teu projeto... Sente-te à vontade em ler as nossas últimas publicações sobre o Strapi. Estamos a dar o nosso melhor para melhorar o produto, baseando-nos no teu feedback.",
   "app.components.HomePage.welcomeBlock.content.issues": "problemas.",
   "app.components.HomePage.welcomeBlock.content.raise": " ou levante ",
-  "app.components.ImgPreview.hint": "Arraste & solte o seu arquivo nesta area ou {browse} um arquivo para fazer a transferência",
-  "app.components.ImgPreview.hint.browse": "procure",
-  "app.components.InputFile.newFile": "Adicionar um novo arquivo",
-  "app.components.InputFileDetails.open": "Abrir numa nova aba",
+  "app.components.ImgPreview.hint": "Arraste & solte o seu arquivo nesta area ou {browse} um arquivo para o carregar",
+  "app.components.ImgPreview.hint.browse": "escolha",
```

Comment:
Hey just a heads up that this entry is still in formal Portuguese while the rest is in informal Portuguese.

Reply:
I'm not sure what's better, whether formal or informal.
I've gone with informal for the most part, but it looks like some formal strings slipped through.
What's your opinion on this?

Reply:
I'll always go for informal, but some people who use this are like company owners who are given Strapi by a developer, so maybe formal is a safer bet?

Reply:
@ptalmeida Feel free to open a PR to fix this

Reply:
I think this should be decided in a standardised way and put on [CONTRIBUTING.md](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md), what do you think? 

Since this isn't a concern in English maybe opening an issue on it would be best

Reply:
Yup, sounds good, do you guys want me to do it? However you want

Reply:
Thanks, go ahead LGTM!